lib/loader/loader.js

```diff
@@ -252,7 +252,11 @@ var loader, define, requireModule, require, requirejs;
     this.name = path;
   }
 
-  define.alias = function(path) {
+  define.alias = function(path, target) {
+    if (arguments.length === 2) {
```

Comment:
the upside of relying on arity is that an accidental `undefined` doesn't change behavior.